web/public/templates/common/header.dust

```diff
@@ -22,6 +22,7 @@
           <li><a class="cd-menu__profile-link">{@i18n key="My Profile"/}</a></li>
           <li><a href="/dashboard/my-dojos">{@i18n key="My Dojos"/}</a></li>
           <li><a href="/dashboard/dojos/events/user-events">{@i18n key="My Events"/}</a></li>
+          <li><a href="/dashboard/profile/children">{@i18n key="My Children"/}</a></li>
```

Comment:
Going to make an update to cd-menu in cd-common to only show this to people who actually have children (until we add the ability to add children from this page). Also need to update cd-theme and the nodebb theme with this too.